docs/index.rst

```diff
@@ -309,6 +309,48 @@ Users can pass along the parent_trace_id and parent_span_id via whatever method
 Advanced Usage
 --------------
 
+Trace Filtering
+~~~~~~~~~~~~~~~
+
+It is possible to filter or modify traces before they are sent to the agent by
+configuring the tracer with a processing pipeline. For instance to filter out
+all traces of incoming requests to a specific url::
+
+    processing_pipeline = [FilterRequestsOnUrl(r'http://test\.example\.com')]
```

Comment:
I think our example code can become:
```python
Tracer.configure(settings={
    'PROCESSING_PIPELINE': [
        FilterRequestsOnUrl(r'http://test\.example\.com'),
    ],
})
```
just to make it simpler. Also, as we discussed, this is a WIP functionality so very likely `PROCESSING_PIPELINE` will be named differently.